src/runtime_src/core/pcie/linux/pcidrv.h

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright (C) 2022 Advanced Micro Devices, Inc. All rights reserved.
+
+#ifndef _XCL_PCIDRV_H_
+#define _XCL_PCIDRV_H_
+
+#include "pcidev.h"
+
+// Forward declaration
+namespace pcidev {
+class pci_device;
+};
+
+namespace pcidrv {
+
+class pci_driver
+{
+public:
+  virtual
+  const std::string
+  name(void) const = 0;
+
+  virtual
+  bool
+  is_user(void) const = 0;
```

Comment:
Someone is implementing these, do you want to comment what they are and what they are suppose to return?

Reply:
Yes, will add comment.